### ISSUE TITLE
getting error when run JS client side

### ISSUE THREAD

**Issue by halhwadi** [jazzband/django-push-notifications#629]
When running the JS code provided in read me section i got the below error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'match')
```

lines code caused this issue is:

```javascript
var ua = userAgent, tem, M = ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || [];

 var browser = loadVersionBrowser();
```

**Comment by azmeuk** [comment]
Browser guessing will be deprecated with #674 so this piece of code won't be necessary at all.

**Comment by azmeuk** [comment]
Fixed by #674
